src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb

```diff
@@ -61,6 +61,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
         End Function
 
         <Extension()>
+        Public Function IsNamespaceDeclarationNameContext(syntaxTree As SyntaxTree, position As Integer, cancellationToken As CancellationToken) As Boolean
+            Dim statement = syntaxTree.GetTargetToken(position, cancellationToken).GetAncestor(Of NamespaceStatementSyntax)
+            Return statement IsNot Nothing AndAlso statement.Name.Span.IntersectsWith(position)
```

Comment:
1. I'm not thrilled that this code differs from teh C# version. 
2. I'm not sure how sufficient it would be when the user just types ```Namespace``` and doesn't have a name.  Now you're going to be checking the span of a missing node.  And those are notoriously finicky.

I'm happy to use the span if the name is there.  But if it isn't, we should explicitly make sure the code works for the case it isn't.